test/test_utils/__init__.py

```diff
@@ -280,7 +280,9 @@ def get_dockerfile_path_for_image(image_uri):
 
 def get_expected_dockerfile_filename(device_type, image_uri):
     if is_covered_by_ec2_sm_split(image_uri):
-        if is_ec2_sm_in_same_dockerfile(image_uri):
+        if "graviton" in image_uri:
```

Comment:
Thanks for adding this. Can we do the following:
1. Add logic in method `is_covered_by_ec2_sm_split` [link](https://github.com/aws/deep-learning-containers/blob/5ae80ebc2ad3f6723b53f0ab7bf8fe8172851681/test/test_utils/__init__.py#L455) to return `False` if the image is `Graviton`
2. Add a condition after [this](https://github.com/aws/deep-learning-containers/pull/2282/files#diff-5d5511865c124cd43c10b4a8f75f026de33c01632acdee7f0fbc849ae5498eb1R299) for Graviton images

Reply:
I think we still want "is_covered_by_ec2_sm_split" logic to remain as an option for Graviton SageMaker images. if setting it to false when graviton is used, would pipeline not fine the Graviton Sagemaker images?